server/@types/make-recall-decision-api/models/PpudUser.ts

```diff
@@ -0,0 +1,4 @@
+export type PpudUser = {
+  userName: string,
```

Comment:
Field name mix up corrected, however  I'm going to leave this in.

Reply:
`fullName` should have replaced `userName`, not `teamName`

Reply:
argh! this got through the testing

Reply:
I see you've corrected this file now. However, you said it got past the testing, so it's making me wonder why the testing didn't catch it - is there some test that should've been change, or that is missing something?

Reply:
Yer, we could look to implementing some runtime type checking, perhaps something like Zod? I've never used it before but it would have caught this typo.
Not for this release tho :)